libparsers/src/utils.cpp

```diff
@@ -338,5 +338,14 @@ namespace util {
 
 		return contents;
 	}
+
+	bool is_numeric(std::string const& str) {
+		return std::all_of(str.begin(), str.end(), [](char c) {
```

Comment:
Why can't you just use parse_loose_number here and throw away the result. It will verify that it is actually a number.

Reply:
This question still stands.